docs/docs/usage.md

```diff
@@ -595,7 +595,7 @@ The above syntax is also supported in `deps`.
 
 NOTE: If you want to call a task declared in the root Taskfile from within an
 [included Taskfile](#including-other-taskfiles), add a leading `:` like this:
-`task: :task-name`.
+`task::task-name`.
```

Comment:
This was actually correct before.

Reply:
So this is expected behavior?

Reply:
Maybe this example would be clearer with syntax highlighting:

```yaml
tasks:
  my-task:
    cmds:
      - task: :task-in-parent-file
```

Reply:
Ok, I see, thank you.